### ISSUE TITLE
Estimated deaths and death rate for U.S. population

### ISSUE THREAD

**Issue by acflippo** [pr3d4t0r/COVIDvu#183]
Provide a worst scenario case for estimated number of deaths and death rate by gender & age based on current China's latest death rate applied to the U.S. population gender & age mix.
[Coronavirus_ US Population Death by age & gender.xlsx](https://github.com/pr3d4t0r/COVIDvu/files/4352164/Coronavirus_.US.Population.Death.by.age.gender.xlsx)

This does not take into consideration of existing medical conditions as there are no statistics being published regarding increase death rates.

The take-away is that majority of deaths (if US is affected in the same way as China) then people aged 60+ will have the highest death rate which we already know but this helps quantify it a little better.

**Comment by kinabalu** [comment]
@acflippo thank you for sharing that with us, somewhat sobering to look at those projections.

@jaryaman @pr3d4t0r inside the excel @acflippo has provided sources and a view into how that graph is put together. what do you guys think about getting something like this on the website?

**Comment by jaryaman** [comment]
@kinabalu @acflippo Yep, looks good to me. I will self-assign and add it to my stack. 

Shall we integrate into `COVIDvu-predict.ipynb` for starters, to get a feel for how to visualize? We could simply make the plot in the spreadsheet for a selection of countries in plotly. 

And yes, those are grim statistics.

**Comment by pr3d4t0r** [comment]
Looking at it now/today.  Thank you!  I'll give it a higher priority.

**Comment by acflippo** [comment]
Yes, the maximum death numbers are sobering so I suggest publishing the mortality rate by gender & age.  That will show how deadly the virus is.

**Comment by jaryaman** [comment]
Hmm, looking at this in more detail. The graphical model for variables death (d), age (a), and sex (s) is 

s -> d <- a

which means, amongst the deceased, age is not independent of sex due to the explain away effect ([see here for example](https://www.cl.cam.ac.uk/teaching/1011/L101/ml4lp-lect3.pdf)). 

So you need p(a,s|d) to evaluate p(d|a,s) with Bayes rule. p(a,s|d) is not equal to p(a|d)p(s|d) due to the collider motif in the graphical model. 

I don't see p(a,s|d) in [the primary source](http://weekly.chinacdc.cn/en/article/id/e53946e2-c6c4-41e9-9a9b-fea8db1a8f51) only p(a|d) and p(s|d).

@acflippo, any thoughts?
